src/main/java/nl/recognize/dwh/application/loader/AbstractEntityLoader.java

```diff
@@ -308,6 +341,11 @@ public void setIdentifier(String idColumn, String identifier) {
             predicates.add(criteriaBuilder.equal(root.get(idColumn), identifier));
         }
 
+        @Override
+        public void setIdentifier(String idColumn, UUID identifier) {
+            predicates.add(criteriaBuilder.equal(root.get(idColumn), identifier));
```

Comment:
Zie andere opmerking...

Reply:
Je kunt nu allebei meegeven, er is nog een functie setIdentifier